### PR TITLE
SMZ3: Fixing minor typos on options

### PR DESCRIPTION
**Pull request by ShineSyndrome**
## What is this fixing or adding?

Fixing typos on option tooltips

### CODE REVIEW

**Comment by alwaysintreble** [comment]
@lordlou

File: worlds/smz3/Options.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class SwordLocation(Choice):
     Randomized - The sword can be placed anywhere.
     Early - The sword will be placed in a location accessible from the start of
     the game.
-    Unce assured - The sword will always be placed on Link's Uncle."""
+    Uncle - The sword will always be placed on Link's Uncle."""
```
Comment (LegendaryLinux) [inline worlds/smz3/Options.py:28]: Should this be `Uncle Assured`?

Comment (ShineSyndrome) [reply worlds/smz3/Options.py:28]: The option itself is "Uncle", makes more sense to match. Probably fine either way.

**Review by nicholassaylor — APPROVED** [review]
Changes LGTM

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**